packages/arb-bridge-eth/contracts/rollup/IRollupCore.sol

```diff
@@ -71,7 +73,16 @@ interface IRollupCore {
         uint256 finalBalance
     );
 
-    function getNode(uint256 nodeNum) external view returns (INode);
+    /**
+     * @notice Get the Node properties for the given index. 
+     * We can't expose the full Node publicly as it contains a mapping 
+     */
+    function getNodeProps(uint256 nodeNum) external view returns (NodeProps memory);
+
+    /**
+     * @notice Check if the specified node has been staked on by the provided staker
+     */
+    function nodeHasStaker(uint256 nodeNum, address staker) external view returns(bool);
```

Comment:
`returns(bool)` missing a whitespace in there.
Do you have the precommit linter configured?